lib/Doctrine/DBAL/Connection.php

```diff
@@ -199,99 +199,49 @@ public function __construct(
      *
      * @return mixed[]
```

Comment:
I probably would have changed this to `array<string, mixed>`. Whether that is right or not is another question :)

Reply:
It definitely would be a more proper annotation.